revup/github_utils.py

```diff
@@ -323,13 +323,13 @@ async def query_everything(
         this_node = pr_result["data"]["repository"][user_id_out[i]]
         if len(this_node["nodes"]) == 0:
             logging.warning("No matching user found for {}".format(user_id))
-        elif this_node["totalCount"] > len(this_node["nodes"]):
-            logging.warning("Too many matching users found for {}".format(user_id))
         else:
+            if this_node["totalCount"] > len(this_node["nodes"]):
+                logging.warning("Too many matching users found for {}".format(user_id))
```

Comment:
Will this warn if there are multiple substring matches but only one prefix match?

Reply:
this warning is for  when github tells you there are more matches than it can return (max return is set to 25 atm). this would mainly happen if you there were more than 25 aarons, you wouldn't be able to match against a later alphabetical one without being more specific

Reply:
one warning that would be new that i'm not handling here is if none of the 25 returned names from github actually prefix match. in that case we would just take the first one rather than the shortest one. i think this is fine for now, but i'll add a warning for this case

Reply:
Yeah that seems fine